multiple_generate.sh

```diff
@@ -7,19 +7,18 @@ function clean() {
 function cloner() {
   URL_SPLIT=(${1//// })
   REPONAME=(${URL_SPLIT[3]//./ })
-  git clone $1 ${REPONAME}_temp &
+  git clone $1 ${REPONAME} &
 }
 
-function repoCleaner() {
+function genSubIndex() {
   URL_SPLIT=(${1//// })
   REPONAME=(${URL_SPLIT[3]//./ })
-  cd ${REPONAME}_temp
-  mkdir ../$REPONAME
-  cp -a docs/. ../$REPONAME/.
+  cd $REPONAME
+  python $BASE/modules/scripts/genindex.py ./
```

Comment:
I have changed to run the script on doc folder. and also made some changes on py file please check that also
sample generated toc tree
```
.. toctree::
   :maxdepth: 1
   :caption: Tempcheck

   tempCheck/docs/index
.. toctree::
   :maxdepth: 1
   :caption: Stockiumai

   stockiumAI/docs/index
```